driver_cpl/driver/prep_ocn_mod.F90

```diff
@@ -1192,18 +1198,26 @@ subroutine prep_ocn_calc_r2x_ox(timer)
     !
     ! Local Variables
     integer :: eri
-    type(mct_avect), pointer :: r2x_rx
-    character(*), parameter  :: subname = '(prep_ocn_calc_r2x_ox)'
+    type(mct_avect), pointer   :: r2x_rx
+    character(len=shr_kind_cs) :: fldlistl, fldlisti
+    character(*), parameter    :: subname = '(prep_ocn_calc_r2x_ox)'
     !---------------------------------------------------------------
 
     call t_drvstartf (trim(timer),barrier=mpicom_CPLID)
+    if (lwiso_runoff) then
+      fldlistl='Forr_rofl:Forr_rofl_16O:Forr_rofl_18O:Forr_rofl_HDO'
```

Comment:
Yeah, I don't know how to get around that. I thought about getting fancy and setting

```
fldlistl='Forr_rofl'
```

and then appending the other three fields if `lwiso_runoff` is true, but that's still hard-coding in the field names. (Though it would be a preferable construct if some future option that is independent of the water isotopes could also impact what runoff fields are mapped.)